### DISCUSSION TITLE
Introductions

### DISCUSSION THREAD

**Discussion by vanessaklee** [vanessaklee/elixir_conf22#12] [Q&A]
### Welcome to ElixirConf 2022!

This is an Introduction to Machine Learning in Elixir with Nx and Axon!

We are excited you chose to join us for this training. Take a moment to introduce yourself here.

**Comment by n0um3n4** [comment]
Hi, this is me. I'm exciting to take this training!! 🎉 
 
I majored in physics (for better or worst) but I live off of coding in Elixir and Rust.
Feel free to reach me 😃

**Comment by alde103** [comment]
Hi, I'm an Embedded Systems Engineer looking forward for the training :)

**Comment by nzakariagp** [comment]
Hello, I'm new to the Elixir space altogether. Our company is transitioning to using the Elixir toolchain as our de facto technology stack. 
Interesting in learning about Nx and Axon.

**Comment by spunkedy** [comment]
Hello!

I have been using elixir for about 6 years, but not with any ML.

**Comment by arubis** [comment]
👋 Hi all! I'm Dylan, and have been working adjacent to (and occasionally with) the Elixir ecosystem mostly on the side for some years.  I help organize the local Denver Elixir/Erlang meetup & would love to get you in front of a friendly audience!

My exposure to Nx and Axon is limited to excitedly reading blog posts.

I'm based locally in Denver and have been here for almost a decade.

**Comment by afhammad** [comment]
Hi, Ahmad @ [LiveFlow](https://www.liveflow.io) based in London.

1 year into using Elixir coming from a Clojure/Ruby background. 

ML novice.

**Comment by trodrigu** [comment]
Hello, I'm Thomas/Tommy!  I'm an ML beginner.  Love Elixir and Rust
